mews-api.md

```diff
@@ -813,13 +856,23 @@ _¹ It is required that the code remains the same within each booking modificati
 
 #### Extra
 
+* Total cost of the extra product should be sent in `net` or `gross` amounts. Either both `gross` and `net` amounts, or one of them should be sent.
```

Comment:
This says sending `gross`, `net`, or both are supported. Why is this different from line 568?